gecko/Dockerfile

```
@@ -7,38 +7,38 @@ RUN __RR_VERSION__="5.3.0" \
  && sudo dpkg -i rr.deb \
  && rm -f rr.deb
 
-# Install Firefox build dependencies.
+# Install Firefox build dependencies, including moz-phab
 # One-line setup command from:
-# https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/Linux_Prerequisites#Most_Distros_-_One_Line_Bootstrap_Command
+# https://firefox-source-docs.mozilla.org/setup/linux_build.html#one-line-bootstrapping
 ENV SHELL=/bin/bash
 RUN sudo apt-get update \
- && git clone https://github.com/mozilla/gecko-dev/ /tmp/gecko \
+ && git clone --progress https://github.com/mozilla/gecko-dev/ /tmp/gecko \
  && cd /tmp/gecko \
  && ./mach bootstrap --no-interactive --application-choice=browser \
  && sudo apt-get remove -y nodejs nodejs-doc \
  && sudo rm -rf /tmp/gecko /var/lib/apt/lists/*
 
+# Install moz-phab
+ENV SHELL=/bin/bash
+RUN sudo apt-get update \
+ && pip3 install MozPhab \
+ && echo "PATH=\"\$PATH:$HOME/.local/bin/moz-phab \"" >> $HOME/.bashrc \
```

I'm not sure the space is useful here:

```suggestion
 && echo "PATH=\"\$PATH:$HOME/.local/bin/moz-phab\"" >> $HOME/.bashrc \
```

---

Ah wait, it seems that `/home/gitpod/.local/bin` doesn't even exist.

But in Gitpod, I can already find `moz-phab` here:

```
$ which moz-phab
/home/gitpod/.pyenv/shims/moz-phab
```

So maybe let's just remove this line:

```suggestion
```